### ISSUE TITLE
Are there errors in your file ? I met error when extract them

### ISSUE THREAD

**Issue by Kyridiculous2** [facebookresearch/Replica-Dataset#42]
Any ideas on how to solve this ? I successfully extracted part of datasets

![image](https://user-images.githubusercontent.com/26832000/70292848-54d9d600-181a-11ea-921a-3c5ca7b18113.png)

**Comment by jstraub** [comment]
Did you use the download and extract script? Can you verify that all dataset parts have finished downloading?

**Comment by Kyridiculous2** [comment]
Yes,  if the --continue opotion can ensure that it has been completely downloaded
I can successfully extract a few envs till this frl_apartment_1, I found it maybe in parti/h then I redownload that part but the same problem remains.....
Could  you offer me other download options? I'm planning to use your dataset for the coming conference due to the high quality, so it's a little bit hurry....

**Comment by ventusff** [comment]
hi @Kyridiculous2 , there's no error in their files, so it must be something wrong when your download.
a md5sum list may be of help for you and all others who enconter this file error problem:
```
f3eaa7c452055e4ac788bf1f4d47f4cc  replica_v1_0.tar.gz.partaa
a6ea3efb76588dd50923205003e6623f  replica_v1_0.tar.gz.partab
07fe2eec5d3eab7686b0c0dbb26ab2dc  replica_v1_0.tar.gz.partac
bcbeee782b78d9052774593306498106  replica_v1_0.tar.gz.partad
54bd3f1bbd81e39a3fe0d5cee65f0981  replica_v1_0.tar.gz.partae
c3c485960ecbb49ce2fdb183865556d7  replica_v1_0.tar.gz.partaf
553ec538c1dc156d421e1b541af6de6a  replica_v1_0.tar.gz.partag
ad04202298da63343d6fc41bc1ae2ece  replica_v1_0.tar.gz.partah
e34a7f85f49c2c1d05ac0184a3c4dca2  replica_v1_0.tar.gz.partai
c9d12878dc78c823beaf75a9218da79d  replica_v1_0.tar.gz.partaj
54bd1bb0049d62e686c6764954d049e1  replica_v1_0.tar.gz.partak
3e59749231ce367263784489adbcc439  replica_v1_0.tar.gz.partal
9235db1a79d3d4b5dffb38e1d2066d20  replica_v1_0.tar.gz.partam
31f404fb30559808be18734e14ba972e  replica_v1_0.tar.gz.partan
376a75bdb73b59ea293d0af844f541f8  replica_v1_0.tar.gz.partao
1a233e4549713867a40df261c49c9169  replica_v1_0.tar.gz.partap
2e8ae8805b4d6fec7faa75dc91d17d7c  replica_v1_0.tar.gz.partaq
```

**Comment by richagadgil** [comment]
Hi, I'm experiencing the same problem, was this ever resolved?

**Comment by zhangchuanyi96** [comment]
After downloading for countless times, I finally managed to solve this problem.

First, write **md5sum** in files, for example in replica_v1_0.tar.gz.partaa.md5:
`f3eaa7c452055e4ac788bf1f4d47f4cc  replica_v1_0.tar.gz.partaa`

Then use a script to automatically download files and check md5:
`
#!/usr/bin/env bash

# Abort on error
set -e

for p in {a..q}
do
  while ! md5sum -c replica_v1_0.tar.gz.parta$p.md5
  do
    # Ensure files are continued in case the script gets interrupted halfway through
    rm replica_v1_0.tar.gz.parta$p
    wget --continue https://github.com/facebookresearch/Replica-Dataset/releases/download/v1.0/replica_v1_0.tar.gz.parta$p
  done
done
`
